recipes/webcolors/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.5" %}
+
+package:
+  name: webcolors
+  version: {{ version }}
+
+source:
+  fn: webcolors-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/webcolors/webcolors-{{ version }}.tar.gz
+  md5: 2e9da1bbb7a087c6108e05be64640afb
+
+build:
+  number: 0
+  script:
+    - source activate _build  # [unix]
+    - activate _build         # [win]
```

Comment:
Good question. If I had a working Windows machine, I could tell you. 😉 Unfortunately, no such luck. 😞